scripts/compile_proto.sh

```diff
@@ -7,7 +7,7 @@ mkdir -p proto/js
 for filepath in proto/*.proto; do
   filename=$(basename "$filepath" .proto)
   # Compiles JavaScript
-  pbjs -t static-module -w commonjs --no-beautify --no-convert --no-verify --no-delimited -o proto/js/$filename.js $filepath
+  pbjs -t static-module -w commonjs --no-beautify --no-convert --no-verify -o proto/js/$filename.js $filepath
```

Comment:
Might need to be part of our `default.nix`?

At what stage of the pipeline are we compiling protobuf files? Every time it changes right? So it's a dev tool. Then it maybe part of the `npm run` scripts? And then during compilation? But we are committing the generated js/ts source files to git right?

Reply:
Yeah I guess we can do it in our `default.nix` and it is also already a part of our `npm run` scripts. At the moment we are committing the generated js/ts files but we can just gitignore them if you think that's the way to go :+1: